docs_src/dev-tools/run_camera_simulator.md

```diff
@@ -4,6 +4,20 @@
 
 If you do not have a camera device plugged into the system, run the camera simulator to view the pipeline analytic results based on a sample video file to mimic real time camera video. You can also use the camera simulator to infinitely loop through a video file for consistent benchmarking. For example, if you want to validate whether the performance is the same for 6 hours, 12 hours, and 24 hours, looping the same video should produce the same results regardless of the duration.
 
+## Supported Input Videos
+
+The following simulated camera videos and their respective video outputs are listed in the table below. These are based on an index number in the script that downloads the sample videos. The camera numbers may change if custom videos are added.
```

Comment:
The camera numbers may change if custom videos are added?  I thought the purpose of this PR is to make sure the camera number won't change when custom videos will be added in future; otherwise the indexing seems defeats this purpose.  I agree that the total number of cameras can be changed but not the camera sequence number.

Reply:
Correct, the purpose of this PR is to ensure the order of the videos won't change even if adding a new one.
So, I suggest we change the text to say that adding a new video, will require to follow the index order for the last parameter.